docs/build/getting-started/authenticating-with-lit.md

```diff
@@ -1 +1,43 @@
-# Authenticating with Lit
\ No newline at end of file
+# Authenticating with Lit
+After connecting, sending requests to the Lit network requires authentication. This authentication is necessary to ensure that the Lit network can identify your session and only grant you access to the resources you're authorized to access.
```

Comment:
Consider "In order to send transactions on Lit, you need to authenticate with the Lit nodes. The authentication process allows you to establish an active session and ensures that you have access to the capabilities relevant to your specific use case"